relay-server/src/services/processor/span/processing.rs

```diff
@@ -342,10 +358,14 @@ fn normalize(
             .collect(),
     );
 
-    if let Some(measurements) = span.measurements.value_mut() {
-        // TODO: find the browser name by normalizing user-agent
-        normalize_span_performance_score("".to_string(), measurements, performance_score);
-    }
+    let mut event = Event {
+        contexts,
+        measurements: span.measurements.clone(),
+        spans: Annotated::from(vec![Annotated::new(span.clone())]),
+        ..Default::default()
+    };
```

Comment:
This turned out hackier than I thought, with all the cloning. We could instead create the pseudo-event before the `retain_items` loop, pass that into `normalize()` instead, and `.take()` measurements & spans in and out of that to prevent cloning. But I would not invest time into that, we should instead have a generic `normalize_performance_score` function that works on both spans and events.